README.md

```diff
@@ -2,14 +2,22 @@ A wrapper for running GDAL in the browser using [gdal-js](https://github.com/ddo
 
 ![](https://github.com/azavea/loam/workflows/Tests/badge.svg)
 
-# Developing
+# Installation
+```
+npm install loam
+```
 
-1. `yarn install`
-2. `yarn dev` and in another session `yarn test:watch`
+Assuming you are using a build system, the main `loam` library should integrate into your build the same as any other library might. However, in order to correctly initialize the Emscripten environment for running GDAL, there are other assets that need to be accessible via HTTP request at runtime, but which should _not_ be included in the main application bundle. Specifically, these are:
+
+- `loam-worker.min.js`: This is the "backend" of the library; it initializes the Web Worker and translates between the Loam "frontend" and GDAL.
+- [`gdal.js`](https://www.npmjs.com/package/gdal-js): This initializes the Emscripten runtime and loads the GDAL WebAssembly.
+- [`gdal.wasm`](https://www.npmjs.com/package/gdal-js): The GDAL binary, compiled to WebAssembly.
+- [`gdal.data`](https://www.npmjs.com/package/gdal-js): Contains configuration files that GDAL expects to find on the host filesystem.
 
-Built libraries are placed in `lib`.
+All of these files will be included in the `node_modules` folder after running `npm install loam`, but it is up to you to integrate them into your development environment and deployment processes.
 
-# Basic usage
+# API Documentation
+## Basic usage
 
 ```javascript
 // Load WebAssembly and data files asynchronously. Will be called automatically by loam.open()
```

Comment:
This example, where `loam.initialize()` is called but the promise isn't handled, feels a bit more than basic to start with. What happens if `loam.open()` is called before the `loam.initialize()` promise completes? Will we attempt to load twice or will it just work? For clarity, it might be worth omitting the initialize call from the basic example. The initialize method is still in the API function list below and the note about it being sometimes helpful to call this early could be added there, specifying how it behaves when called more than once.

Reply:
I don't feel strongly about omitting the `loam.initialize()` call but was going to mention adding the note about benefits of calling it early to the API documentation.

Reply:
I decided to keep `initialize()` in the example; although it's more advanced usage, all of our projects that have used Loam so far have eventually ended up calling `initialize()`, so I want to keep it in the quick-start because I think it'll end up being useful for most people eventually.

I will expand on the section for `initialize()` in the API docs, that's a great suggestion.